src/api/app/views/webui/user/user_profile_redesign/_info.html.haml

```diff
@@ -0,0 +1,68 @@
+.card.mb-3
+  .card-body
+    .row.mb-3.mb-md-0
+      .col-4.col-sm-2.col-md-12.text-center
+        = image_tag_for(user, size: 100)
+      .col-8.col-sm-10.col-md-12.pl-0.p-md-3
+        %h4#home-realname
+          = user.realname
+        %h5.text-muted#home-login
+          = user.login
+
+    %p= render_as_markdown(user.biography)
+
+    - role_titles.each do |title|
+      %span.badge.badge-secondary
+        = title.upcase
+
+    .mt-4
+      - if User.session
+        = mail_to(user.email, title: "Send mail to #{user.name}", class: 'd-block') do
+          %i.fas.fa-envelope.mr-1
+          = user.email
+
+      - if policy(user).rss_access?
```

Comment:
To follow up on my [previous comment](https://github.com/openSUSE/open-build-service/pull/10295#discussion_r507557696), here's how we could check if we need to show the `RSS for Notifications` link or not.

```suggestion
      - if if user.rss_token && @is_displayed_user
```

Reply:
Thanks. Done!